src/utils/API_KEY.ts

```diff
@@ -0,0 +1 @@
+export const API_KEY: string = '9cfd40b43fdd0d0687798042ba3050eb';
```

Comment:
NOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOO
Man never do it!
API key must be saved in `.env` files